cirq-google/cirq_google/line/placement/anneal.py

```diff
@@ -30,14 +31,14 @@
 class AnnealSequenceSearch:
     """Simulated annealing search heuristic."""
 
-    def __init__(self, device: 'cirq_google.XmonDevice', seed=None) -> None:
+    def __init__(self, device: 'cirq_google.GridDevice', seed=None) -> None:
         """Greedy sequence search constructor.
 
         Args:
           device: Chip description.
           seed: Optional seed value for random number generator.
         """
-        self._c = device.qubits
+        self._c = cast(Set[cirq.GridQubit], device.metadata.qubit_set)
```

Comment:
Does `GridDevice` not define its `qubit_set` as containing `GridQubit`? I guess this is fine for now, but would be nice to fix in the future.